map/query-features-with-arcade-expression/src/main/java/com/esri/samples/query_features_with_arcade_expression/QueryFeaturesWithArcadeExpressionSample.java

```diff
@@ -0,0 +1,197 @@
+                identifyLayerResultFuture.addDoneListener(() -> {
+                  if (identifyLayerResultFuture.isDone()) {
+                    try {
+                      var identifyLayerResult = identifyLayerResultFuture.get();
+
+                      // only execute query if the police beats layer has data at the selected point
+                      if (identifyLayerResult.getElements().size() != 0) {
+                        ArcGISFeature feature = (ArcGISFeature) identifyLayerResult.getElements().get(0);
+
+                        // show callout and text without data
+                        mapView.getCallout().setTitle(calloutText);
+                        mapView.getCallout().setDetail("...");
+                        mapView.getCallout().showCalloutAt(mapPoint);
+                        mapView.getCallout().setVisible(true);
+
+                        // setup arcade expression and evaluator
+                        String arcadeExpressionString = "var crimes = FeatureSetByName($map, 'Crime in the last 60 days');\n" +
+                          "return Count(Intersects($feature, crimes));";
+                        var arcadeExpression = new ArcadeExpression(arcadeExpressionString);
+                        var arcadeEvaluator = new ArcadeEvaluator(arcadeExpression, ArcadeProfile.FORM_CALCULATION);
+
+                        // instantiate key/value pairs
+                        var hashMap = new HashMap<String, Object>();
+                        hashMap.put("$feature", feature);
+                        hashMap.put("$map", mapView.getMap());
+
+                        // evaluate the arcade expression asynchronously
+                        ListenableFuture<ArcadeEvaluationResult> resultFuture = arcadeEvaluator.evaluateAsync(hashMap);
+
+                        // wait for the expression to finish evaluating before attempting to access it
+                        resultFuture.addDoneListener(() -> {
+                          if (resultFuture.isDone()) {
+                            try {
+                              // get result from async method and cast to int
+                              var arcadeEvaluationResult = resultFuture.get();
+                              var crimesCount = Math.round((double) arcadeEvaluationResult.getResult());
+
+                              // add data from arcade evaluation to callout
+                              mapView.getCallout().setDetail(String.valueOf(crimesCount));
+                            } catch (Exception e) {
+                              throw new RuntimeException(e);
+                            }
+                          }
+                        });
+                      } else {
+                        mapView.getCallout().setVisible(false);
+                      }
+                    } catch (Exception e) {
+                      throw new RuntimeException(e);
+                    }
+                  }
+                });
+            }
```

Comment:
something's off with the formatting here: if you do `ctrl-alt-l` (lower case L) IntelliJ will auto correct format for you